internal/db/nosql/cassandra/cassandra.go

```diff
@@ -266,18 +267,24 @@ func (c *client) Query(stmt string, names []string) *Queryx {
 	return gocqlx.Query(c.session.Query(stmt), names)
 }
 
+func clientError(err error, keys ...string) error {
+	switch err {
+	case ErrNotFound:
+		return errors.Wrap(err, errors.ErrCassandraNotFound(keys...).Error())
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
It you wrap the original error with error string of newly generated one, you cannot identify your error in both `IsErrCassandraNotFound` and `IsErrCassandraUnavailable`.
In this case, the original one should be stringify.